twyla/kubedeploy/__init__.py

```diff
@@ -200,6 +201,32 @@ def info(name: str, namespace: str):
     kube.info()
 
 
+@cli.command()
+@click.option('--namespace', help='Namespace in the cluster.',
+              default='default')
+@click.option('--group',
+              help='Value of the servicegroup selector to select by.',
+              default='twyla')
+def cluster_info(group: str, namespace: str):
+    kubectl = Kubectl()
+    kubectl.namespace = namespace
+
+    state = kubectl.list_deployments(selectors={'servicegroup': group})
+
+    for item in state['items']:
```

Comment:
It would be great to have tests for the output format, as it will be used to recreate the state in the next step. Testing would be much easier if you just put the for loop into a function and tested that.

Reply:
The tests for the relevant output formats are in the next PR: https://github.com/TwylaHelps/twyla.kubedeploy/commit/6dbaa28d2e66f6b07053ac64e430604383d62525
I can add basic tests for the interactive output as well.. (basically assertions the printer was called correctly)

Reply:
They actually are. In `test_commands.py`.